tests/common/include/tests/common/material_interaction.inl

```diff
@@ -308,12 +308,9 @@ TEST(material_interaction, telescope_geometry_energy_loss) {
     // It is not perfectly precise as the track loses its energy during
     // propagation. However, since the energy loss << the track momentum,
     // the assumption is not very bad
-
-    // The navigation starts on the first surface, which is therefore not
-    // accounted
```

Comment:
It is true that the navigator starts on the first surface, but are you sure that it is not accounted for the material interaction? I am asking because I was also not sure

Reply:
I believe you are right. I thought this was the case because the navigator should skip a surface that it is already on during volume initialization in order to avoid double counting portals, but it looks like it actually correctly flags ```on_module``` in this case....